fake.go

```diff
@@ -61,6 +64,11 @@ type FakeTable struct {
 	Maps map[string]*FakeMap
 }
 
+// FakeFlowtable wraps Flowtable for the Fake implementation
+type FakeFlowtable struct {
```

Comment:
(This isn't actually needed since Flowtables don't have any "contents", but I guess it makes sense to add it just for consistency.)

Reply:
ack